google-cloud-util/google-cloud-compat-checker/src/main/java/com/google/cloud/compatchecker/GoogleCloudCompatChecker.java

```diff
@@ -43,30 +44,96 @@ protected void logProperty(String s, String s1) {
   }
 
   public static void main(String[] args) {
+    Boolean checkResult = check();
+    if (checkResult != null && !checkResult) {
+      System.exit(1);
```

Comment:
I agree with codacy-bot. This looks more like a bash script now. Why not just throwing an exception with a meaningful message?